scripts/mk_repo_file.py

```diff
@@ -21,8 +21,8 @@
 
 REPO_FLAGS = [
     "--quiet",
-    "--repo-url=https://github.com/danielfullmer/tools_repo",
-    "--repo-rev=9ecb9713ee5adba95120acbc0bfef1c77b02637f",
+    "--repo-url=https://github.com/jaen/tools_repo",
+    "--repo-rev=dca531f6d6e9fdcf00aa9d18f0153bd66a2e32ea",
```

Comment:
That sounds reasonable,I'll try implementing it like this

Reply:
Turns out that doesn't help much, because the regular `gitRepo` derivation is basically just a launcher (c.f. https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/version-management/git-repo/default.nix#L32) that will then download the missing code from the interwebs (c.f. https://github.com/jaen/tools_repo/blob/main/repo#L669-L674). While it would be nice to make it reproducible, I'd say it's maybe out of scope for a simple test fix?

Reply:
Did overlay v2.45 in pkgs, though. Not sure if there's any downside to those versions not being the same, but might as well have them match.